src/main/java/org/truffleruby/cext/CExtNodes.java

```diff
@@ -307,6 +307,16 @@ public long longlong2num(long num) {
 
     }
 
+    @CoreMethod(names = "ULL2NUM", onSingleton = true, required = 1)
+    public abstract static class ULL2NUMNode extends CoreMethodArrayArgumentsNode {
+
+        @Specialization
+        public long ulonglong2num(long num) {
+            return num;
+        }
+
+    }
```

Comment:
This should reuse the same logic as below as we convert from unsigned.
Actually, it's enough to modify ruby.c so `rb_ull2inum` calls `ULONG2NUM` since they are both 64-bit unsigned integer types.

Reply:
Fixed, thanks.